docs/native/src/overview.rst

```diff
@@ -1,5 +1,76 @@
 
 ..  _intro:
 
-|stub-icon| Integrating with Office for iOS
-===========================================
+Integrating with Office for iOS
```

Comment:
I think the overview is useful, so I'd be interested in seeing what kind of feedback you hear here.